src/bundle/Resources/views/admin/role/add.html.twig

```diff
@@ -1,39 +1,39 @@
-{% extends "@EzPlatformAdminUi/layout.html.twig" %}
+{% extends 'EzPlatformAdminUiBundle:admin/role:base.html.twig' %}
 
 {% trans_default_domain 'role' %}
 
-{% block breadcrumbs %}
+{% block breadcrumbsAdmin %}
     {% include '@EzPlatformAdminUi/parts/breadcrumbs.html.twig' with { items: [
         { value: 'breadcrumb.admin'|trans(domain='messages')|desc('Admin') },
         { url: path('ezplatform.role.list'), value: 'role.breadcrumb.list'|trans|desc('Roles') },
         { value: 'role.breadcrumb.add'|trans|desc('Creating a new Role') }
     ]} %}
 {% endblock %}
 
-{% block pageTitle %}
+{% block pageTitleAdmin %}
```

Comment:
Underscore block name

Reply:
Yes, I agree. Let's change it to underscore.